android/build.gradle

```diff
@@ -1,56 +1,59 @@
 group 'com.instabug.flutter'
 version '13.0.0'
-
 buildscript {
     repositories {
         google()
         mavenCentral()
+        maven {
+            credentials {
+                username "instabug"
+                password System.getenv()["INSTABUG_REPOSITORY_PASSWORD"]
```

Comment:
if we merge this line to dev,, it will cause build issues if we don't add INSTABUG_REPOSITORY_PASSWORD in system env

Reply:
Deleting the Maven credentials will result in a build failure, as they are used to access the Android custom snapshot for screen loading. We plan to replace it with the Android production version once it's released.